src/Symfony/Component/Form/Extension/Core/Type/ChoiceType.php

```diff
@@ -306,6 +306,7 @@ public function configureOptions(OptionsResolver $resolver)
             'choice_value' => null,
             'choice_attr' => null,
             'preferred_choices' => [],
+            'keep_preferred_choices_order' => false,
```

Comment:
The option name is debatable. This one is a little bit verbose, but tells exactly what it is about.
My alternative is `keep_preferred_order`.